blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/CassandraModel.java

```diff
@@ -256,21 +256,32 @@ public static Granularity getGranularity(String cf) {
         return ALL_COLUMN_FAMILIES;
     }
 
+    public static Collection<String> getAllColumnFamiliesNames() {
+
+        List<String> names = new ArrayList<String>() {{
+                for (ColumnFamily cf : ALL_COLUMN_FAMILIES) {
+                    add(cf.getName());
+                }
+        }};
+        return names;
+    }
+
     public static class MetricColumnFamily extends ColumnFamily<Locator, Long>  {
         private final TimeValue ttl;
 
         public MetricColumnFamily(String name, TimeValue ttl) {
-            super(name, LocatorSerializer.get(), LongSerializer.get());
-            this.ttl = ttl;
+                    super(name, LocatorSerializer.get(), LongSerializer.get());
+                    this.ttl = ttl;
```

Comment:
the indenting here  (and at line 278) is a little weird.  No biggie, but pointing it out in case you touch this again.